script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -1,23 +1,23 @@
 [sha:d155a0f3e2e78032ab2d2a5c68ceeaa849c3823c]{
 -- Page 10009, Offset 15212.
 -- 「わたしの方に来るとはまったく予想外でした。
-"I didn't expect you to come to me.
+"I have to say I'm rather surprised you chose to come to me.
```

Comment:
I have to say, I'm rather surprised you came to me.

Reply:
"came to me" is a little awkward, so don't agree there. Comma is a necessary change, though.